README.md

```diff
@@ -41,66 +45,79 @@ git clone https://github.com/o3de/o3de.git
+*   Wwise audio SDK
+    *   For the latest version requirements and setup instructions, refer to the [Wwise Audio Engine Gem](https://o3de.org/docs/user-guide/gems/reference/audio/wwise/audio-engine-wwise/) reference in the documentation.
 
-### Quick Start Build Steps
+### Quick start engine setup
 
-1.  Create a writable folder to cache 3rd Party dependencies. You can also use this to store other redistributable SDKs.
+To set up a project-centric source engine, complete the following steps. For other build options, refer to [Setting up O3DE from GitHub](https://o3de.org/docs/welcome-guide/setup/setup-from-github/) in the documentation.
+
+1.  Create a writable folder to cache downloadable packages. You can also use this to store other redistributable SDKs.
     
-1.  Install the following redistributables to the following:
-    - Visual Studio and VC++ redistributable can be installed to any location
-    - CMake can be installed to any location, as long as it's available in the system path
+1.  Install the following redistributables:
+    - Visual Studio and VC++ redistributable can be installed to any location.
+    - CMake can be installed to any location, as long as it's available in the system path.
 
-1.  Configure the source into a solution using this command line, replacing <your build path> and <3rdParty cache path> to a path you've created:
+1.  Configure the engine source into a solution using this command line, replacing `<your build path>`, `<your source path>`, and `<3rdParty cache path>` with the paths you've created:
+    ```
+    cmake -B <your build path> -S <your source path> -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<3rdParty cache path>
+    ```
+    
+    Example:
     ```
-    cmake -B <your build path> -S <your source path> -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<3rdParty cache path> -DLY_UNITY_BUILD=ON -DLY_PROJECTS=AutomatedTesting 
+    cmake -B C:\o3de\build\windows_vs2019 -S C:\o3de -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages
     ```
-    > Note:  Do not use trailing slashes for the <3rdParty cache path>
+    
+    > Note:  Do not use trailing slashes for the <3rdParty cache path>.
 
 1.  Alternatively, you can do this through the CMake GUI:
     
-    1.  Start `cmake-gui.exe`
-    1.  Select the local path of the repo under "Where is the source code"
-    1.  Select a path where to build binaries under "Where to build the binaries"
-    1.  Click "Configure"
-    1.  Wait for the key values to populate. Fill in the fields that are relevant, including `LY_3RDPARTY_PATH` and `LY_PROJECTS`
-    1.  Click "Generate"
+    1.  Start `cmake-gui.exe`.
+    1.  Select the local path of the repo under "Where is the source code".
+    1.  Select a path where to build binaries under "Where to build the binaries".
+    1.  Click **Configure**.
+    1.  Wait for the key values to populate. Fill in the fields that are relevant, including `LY_3RDPARTY_PATH`.
```

Comment:
A bit of a heads up here.  If you do it this way, the first time you click Configure on an empty build directory will automatically assume that your 3rdParty path is `$HOME/.o3de/3rdParty`.  This will start downloading large 3p packages to that destination, even if you intend to change it later on with the `LY_3RDPARTY_PATH` cache variable.

It's a bit of a chicken-egg scenario.  You could potentially instruct users to click the `Add Entry` button to add that variable before clicking Configure -- if they intend to use a custom `LY_3RDPARTY_PATH`.

Reply:
Updated with an **Add Entry** step.